### ISSUE TITLE
Implement workflow full text search

### ISSUE THREAD

**Issue by samgj18** [samgj18/workflow-rs#5]
**Is your feature request related to a problem? Please describe.**
As a user I'd like to be able to search through my tags

**Describe the solution you'd like**
Even though this is technically possible by simply piping the list output [after #4 is finished] through [fzf](https://github.com/junegunn/fzf). It'd be nice to have a quick and easy version of this to simply do: `workflow list --search`, and by pressing enter user will go into a `full text search` mode where once found the workflow, he can choose to execute or not.
